aliceO2/NamespaceNamingCheck.cpp

```diff
@@ -23,14 +23,28 @@ namespace aliceO2 {
 const std::string VALID_NAME_REGEX = "[a-z][a-z_0-9]+";
 const std::string VALID_PATH_REGEX = "(.*/O2/.*)|(.*/test/.*)";
 
+const std::string VALID_NAME_REGEX = "[a-z][a-z_0-9]+";
+std::string VALID_PATH_REGEX = "";
+
+NamespaceNamingCheck::NamespaceNamingCheck(StringRef Name, ClangTidyContext *Context)
+    : ClangTidyCheck(Name, Context)
+{
+  VALID_PATH_REGEX = Options.getLocalOrGlobal("CheckPathRegex","");
+	if( VALID_PATH_REGEX == "" )
+	{
+	  fprintf(stderr,"Error: User must provide a .clang-tidy file in any of the parent directories of the source files containing a key-value pair for key \'CheckPathRegex\'");
```

Comment:
Is it true that we need a .clang-tidy file or can the option be passed via the command line?

Reply:
Actually yes, we the option can be provided via command line argument. I will fix the error message.